testdata/dnn/download_models.py

```diff
@@ -1049,6 +1049,36 @@ def get_confirm_token(response):  # in case of large files
         member='PP_OCRv3_DB_text_det/PP_OCRv3_DB_text_det.onnx',
         sha='f541f0b448561c7ad919ba9fffa72ff105062934',
         filename='onnx/models/PP_OCRv3_DB_text_det.onnx'),
+    Model(
+        name='YOLOv5n (ONNX)',
+        url='https://github.com/ultralytics/yolov5/releases/download/v7.0/yolov5n.onnx',
+        sha='90aa4b5f8e1ba19166f214f3e4258e5d1d05070b',
+        filename='yolov5n.onnx'),
+    Model(
+        name='YOLOv8n (ONNX)',
+        url='https://github.com/WanliZhong/opencv_extra_data_backup/blob/main/perf_test_data/models/yolov8n.onnx',
```

Comment:
Is there other source for the model rather than personal GitHub repo?

Reply:
The official repository only release PyTorch model, I have to export onnx model manually. I haven't found other GitHub repo release the `yolov8n.onnx` file

Reply:
Ok got it. Seems like this is a common problem. What do you think about adding a method which could export PyTorch models to ONNX file? So we could download original weights.

Reply:
Please provide a link to the original repo.

Reply:
I found a download link https://bj.bcebos.com/paddlehub/fastdeploy/yolov8n.onnx from project https://github.com/PaddlePaddle/FastDeploy, I will replace it

Reply:
Great, thank you!

Reply:
The model from PaddlePaddle has dynamic input, and we can't parse it correctly, so I plan to use the link from project https://github.com/CVHub520/X-AnyLabeling to replace